src/status_im/acquisition/core.cljs

```diff
@@ -65,7 +69,8 @@
 (fx/defn has-referrer
   {:events [::has-referrer]}
   [{:keys [db] :as cofx} flow-state referrer]
-  (when referrer
+  (when (and referrer
+             (get-in db [:acquisition :contract]))
```

Comment:
Remove this line, we should allow advertiser popup even if the contract is disabled as the advertiser may be without reward